lib/bitcoin/fee/SlidingWindowQuantileStoreInitializer.ts

```diff
@@ -0,0 +1,109 @@
+import ISlidingWindowQuantileStore from '../interfaces/ISlidingWindowQuantileStore';
+import ProtocolParameters from '../ProtocolParameters';
+import RunLengthTransformer from './RunLengthTransformer';
+import QuantileInfo from '../models/QuantileInfo';
+import ValueApproximator from './ValueApproximator';
+
+/**
+ * We want to insert some static data in the quantile db to bootstrap the calculation. This class
+ * encapsulates the functionality to do that.
+ */
+export default class SlidingWindowQuantileStoreInitializer {
+
+  private constructor (
+    private genesisBlockNumber: number,
+    private groupSizeInBlocks: number,
+    private windowSizeInGroup: number,
+    private sampleSizePerGroup: number,
+    private initialQuantileValue: number,
+    private mongoDbStore: ISlidingWindowQuantileStore) {
+  }
+
+  /**
+   * Initializes the database if it is empty.
+   *
+   * @param genesisBlockNumber The genesis block.
+   * @param mongoDbStore The data store.
+   */
+  public static async initializeDatabaseIfEmpty (
+    genesisBlockNumber: number,
+    valueApproximator: ValueApproximator,
+    mongoDbStore: ISlidingWindowQuantileStore) {
+
+    // The quantile value below is what we have decided to be the value for the initial
+    // static value.
+    const quantile = 25000;
+    const normalizedQuantile = valueApproximator.getNormalizedValue(quantile);
+
+    const dataInitializer = SlidingWindowQuantileStoreInitializer.createInstance(
+      genesisBlockNumber,
+      normalizedQuantile,
+      mongoDbStore);
+
+    await dataInitializer.addDataIfNecessary();
+  }
+
+  private static createInstance (
+    genesisBlockNumber: number,
+    initialQuantileValue: number,
+    mongoDbStore: ISlidingWindowQuantileStore) {
+
+    return new SlidingWindowQuantileStoreInitializer(
+        genesisBlockNumber,
+        ProtocolParameters.groupSizeInBlocks,
+        ProtocolParameters.windowSizeInGroups,
+        ProtocolParameters.sampleSizePerGroup,
+        initialQuantileValue,
+        mongoDbStore);
+  }
+
+  private async addDataIfNecessary (): Promise<boolean> {
+    const dbIsEmpty = (await this.mongoDbStore.getFirstGroupId()) === undefined;

```

Comment:
Note by @arvinda: what if the node crashes in the middle. Let's do more checks to make sure that we are in a complete and not partial state.